src/main/java/ladder/domain/line/Ladder.java

```diff
@@ -0,0 +1,44 @@
+package ladder.domain.line;
+
+import ladder.domain.player.Players;
+import ladder.domain.strategy.PointStrategy;
+
+import java.util.List;
+import java.util.Objects;
+import java.util.stream.Collectors;
+import java.util.stream.Stream;
+
+public class Ladder {
+    private final List<LadderLine> lines;
+
+    public Ladder(Players players, Height height, PointStrategy pointStrategy) {
+        this.lines = createLines(players, height, pointStrategy);
+    }
+
+    public static Ladder of(Players players, Height height, PointStrategy pointStrategy) {
+        return new Ladder(players, height, pointStrategy);
+    }
+
+    public List<LadderLine> getLines() {
+        return lines;
+    }
```

Comment:
Collections.unmodifiablelist()를 활용해 read-only로 리턴하여 외부에서 값의 변조를 막도록 하면 좋을 것 같아요 :)
```suggestion
    public List<LadderLine> getLines() {
        return Collections.unmodifiablelist(lines);
    }
```